### PR TITLE
Move more docs to docs folder

### PR DESCRIPTION
**Pull request by cijothomas**
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3795 leftover comments from this too.

### CODE REVIEW

File: docs/trace/customizing-the-sdk/README.md [hunk 1]
```diff
@@ -199,11 +199,63 @@ logging, by supporting `Activity` and `LogRecord` respectively.*
 
 ### Resource
 
-// TODO
+[Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md)
+is the immutable representation of the entity producing the telemetry. If no
+`Resource` is explicitly configured, the
+[default](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#semantic-attributes-with-sdk-provided-default-value)
+is to use a resource indicating this
```
Comment (vishweshbankwar) [inline docs/trace/customizing-the-sdk/README.md:206]:
```suggestion
resource is used to indicate the
```

File: docs/trace/customizing-the-sdk/README.md [hunk 2]
```diff
@@ -199,11 +199,63 @@ logging, by supporting `Activity` and `LogRecord` respectively.*
...
+[Service](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#service).
+The `ConfigureResource` method on `TracerProviderBuilder` can be used to set a
```
Comment (vishweshbankwar) [inline docs/trace/customizing-the-sdk/README.md:208]:
```suggestion
The `ConfigureResource` method on `TracerProviderBuilder` can be used to
```

**Review by vishweshbankwar — APPROVED** [review]

File: src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs [hunk 3]
```diff
@@ -97,15 +97,14 @@ public override ExportResult Export(in Batch<Metric> batch)
...
                         {
-                            bucketsBuilder.Append($"Sum: {sum} Count: {count} Min: {metricPoint.GetHistogramMin()} Max: {metricPoint.GetHistogramMax()} \n");
-                        }
-                        else
-                        {
-                            bucketsBuilder.Append($"Sum: {sum} Count: {count} \n");
+                            bucketsBuilder.Append($"Min: {metricPoint.GetHistogramMin()} Max: {metricPoint.GetHistogramMax()} ");
                         }
 
+                        bucketsBuilder.Append("\n\r");
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs:106]: This looks very strange to me, normally folks would use `\r\n` (CRLF) on Windows.

Comment (reyang) [reply src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs:106]: Curious, is the intention to use https://learn.microsoft.com/dotnet/api/system.environment.newline?

Comment (cijothomas) [inline src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs:106]:
```suggestion
                        bucketsBuilder.Append("\r\n");
```

Comment (Kielek) [inline src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs:106]: Myave AppendLine? Library should handle correct endline for us. 
```suggestion
                        bucketsBuilder.AppendLine();
```

**Review by Kielek — APPROVED** [review]

**Review by reyang — APPROVED** [review]

**Pull request merged**